src/Task/Typo3/Stack.php

```diff
@@ -43,21 +44,19 @@ public function run()
      */
     public function execDbDump($fileName, $parameters = [])
```

Comment:
would be better to escape at this place; so the function can safely be called with all kind of content in the array